.github/labels.yml

```diff
@@ -0,0 +1,49 @@
+# Main categories
+- name: 'breaking'
+  color: 'b60205'
+  description: 'Breaking API change or removal of functionality'
+- name: 'bug'
+  color: 'd73a4a'
+  description: "Something isn't working"
+- name: 'documentation'
+  color: 'a2eeef'
+  description: 'Improvements or additions to documentation'
+- name: 'feature'
+  color: '0075ca'
+  description: 'Added or improved external functionality'
+
+# Guidance
+- name: 'good first issue'
+  color: '90ee90'
+  description: 'Good for newcomers'
```

Comment:
Yeah - Can't say I've ever used this tag, or found it useful. I'd be totally happy on dropping it. 

I'd think that this would be "Good for new employees", but even then, I don't think we're big or specialized enough to really make much use of this tag. 

If we're coaching someone new, it might be better to communicate this by assigning the task to them.

Reply:
@amostergaard Meh... On the one hand, I'd say that on its own it doesn't cause any harm/confusion by existing. But I also know that the more tags you add, the more visual clutter / items to weed through, and the more complicated and/or time-consuming the task of finding and tagging what you need. 

In this case, I agree with you and would vote for simplicity (i.e., dropping it). You've never used it, and in 3 years I've never used it. 👍

Reply:
I'm in favour of dropping it as well. I'll take it out when I can.

Reply:
Adopted